code/lib/instrumenter/src/instrumenter.test.ts

```diff
@@ -29,7 +29,7 @@ class HTMLElement {
   }
 }
 
-delete global.location;
+delete (global as Partial<typeof globalThis>).location;
 // @ts-expect-error (global scope type conflicts)
 global.location = { reload: jest.fn() };
 // @ts-expect-error (global scope type conflicts)
```

Comment:
I think better to do a `ts-expect-error` here as well. We are doing really funky stuff here.